.github/workflows/test_pack_doc.yml

```diff
@@ -102,15 +102,13 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
       - name: Package cli app jar with Gradle
         uses: eskatos/gradle-command-action@v1
-        env:
-          versionTag: ${{ needs.prepare-version-name.outputs.versionTag }}
```

Comment:
Same as mentioned in `docker.yml`. We are still using the versionTag a lot, but mostly for job names. I'm not sure if this is relevant anymore if the validator is responsible for its own versioning now? cc @barbeau

Reply:
Yes, I think we should remove all version tag references from the test_pack_doc GitHub Action? I believe it should be printed to the log as part of the commands shown in the JAR file name?

Reply:
I removed the versionTag references here as well.